### ISSUE TITLE
Add the language field for citations items

### ISSUE THREAD

**Issue by Futur3r** [diegodlh/zotero-cita#222]
**Is your feature request related to a problem? Please describe.**
When exporting citations with the Quickstatements translator, I have to manually change the languages of each "title" (i.e und: to fr:) and "Label" (i.e Len to Lfr), because this info is not stipulated in citation items and not editable through the citation editor.

**Describe the solution you'd like**
**A "language" field in the citation editor** and **the ability to "paste"/modify a field value to multiple items** at once.

For example, selecting multiple items with Maj+Left-click and then doing right-click to show a menu and selecting an option called like "modify values" that opens a window where I could select the name of the field and put a value that will be applied to all the items I've selected. That window could look like the window to modify or create "saved searches".

**Describe alternatives you've considered**
For now I can export citations with the quickstatements translator and edit the data manually in a spreadsheet or I can re-import the citation items back into Zotero in BetterBibTex and edit it manually there (I prefer QS for now).

**Comment by Futur3r** [comment]
And actually, it would be great to also have the ability to modify the values of multiple Zotero at once.

**Comment by Dominic-DallOsto** [comment]
Does a language only apply to the title of a work?

I guess this is also a problem with the Quickstatements translator? Or would it correctly export a Zotero item (not a citation) with the language set?

Depending on the complexity, the ability to apply changes to multiple citations at once could get a bit complicated interface-wise. In such a case I might be inclined to go more in the direction of something like an API endpoint #148 so that you could use your programming tool of choice to make modifications to Cita's data as you see fit. But if this feature would be broadly useful to others, and relatively simple enough to implement, I'd be happy to.

**Comment by Futur3r** [comment]
No, it's the language of the entire work and the QS translator also add the "langue of the work" property.

I've opened an issue on the github of the translator.

I don't think it would be that complicated, really it should look like the window to add/modify "saved search".
And I think to would be only useful for the language field, **maybe just a functionality like the one of Zutilo for copy and pasting tags between items.**
It would be like copying the language value of an item, selecting the target items and pasting to the items selected.

**Comment by Futur3r** [comment]
> Depending on the complexity, the ability to apply changes to multiple citations at once could get a bit complicated interface-wise.
Okay, I just found out, it's supported by [Zutilo](https://github.com/wshanks/Zutilo) for Zotero items, it's not straightforward but it's there !

**Comment by HughP** [comment]
the language field in Zotero is not about the language of the title or the
content. The Language field in Zotero passes a value to the CSL exporter to
tell the CSL which locale to use. For example, use APA citation format for
french. One can still have a french title with in an english citation
locale. There is no field currently in Zotero for language of the metadata
or language of the work.
- Hugh

**Comment by Futur3r** [comment]
Oh wow okay, that's not logical at all to me.
Why not just set the locale for all exports manually when citing the document ? Because, wouldn't that mean that for an English person all Language values would be set to "en" ? So what's the point ?
And like, if the Language value would represent the actual language of the work, wouldn't it be right to cite it with the locale of it's language ?

I'm not using this field like that at all, and frankly it's saving me a lot of time to use it as I do.

**Comment by Futur3r** [comment]
same as #99
